### PR TITLE
Hotfix 7.2.3

### LINKED ISSUE #545: Disable manage rights check when installer are not configured to run
Carved out of #528

## Who's affected

Customers using connection strings with no manage rights

## Symptoms

When using a connection string with no manage rights and disabled installers the endpoint will refuse to start. Manage rights are required to operate an endpoint

## Solution

When installers are not enabled the transport should assume everything is configured properly and just run.

### PR DESCRIPTION
**Pull request by danielmarbach**
Closes #545 and #546 

Announcement https://github.com/Particular/Announcements/pull/139

### CODE REVIEW

**Comment by danielmarbach** [comment]
@yvesgoeleven @SeanFeldman you already reviewed it but please review again

File: src/Transport/Seam/TransportResourcesCreator.cs [hunk 1]
```diff
@@ -17,7 +17,16 @@ public TransportResourcesCreator(ICreateTopology topologyCreator, ITopologySecti
 
         public async Task CreateQueueIfNecessary(QueueBindings queueBindings, string identity)
         {
-            if (resourcesCreated) return;
+            if (resourcesCreated)
+            {
+                return;
```
Comment (SeanFeldman) [inline src/Transport/Seam/TransportResourcesCreator.cs]: Question: would log file contain any sort of information why we're not creating infrastructure? I.e. if someone is looking at the log to determine that topology is not created _because_ installers are disabled.

**Comment by danielmarbach** [comment]
both `TransportResourcesCreator `and `SubscriptionManager` call into the topology creator. The only difference is that the transport resource creator does an assert first before it tries to create the topology.

File: src/Transport/Seam/TransportResourcesCreator.cs (hunk 1 above)
Comment (danielmarbach) [inline src/Transport/Seam/TransportResourcesCreator.cs]: Create queues only gets called when the installers are enabled. The assumption of the code is that although the implementation can be called multiple times the queue bindings will contain all the bindings to create and therefore we only create the topology once. I don't think we need to change this as part of the hotfix.

**Comment by SeanFeldman** [comment]
@danielmarbach do you need a hand with releasing this one?

**Comment by danielmarbach** [comment]
@SeanFeldman I can manage it. I was waiting on more feedback on the announcement but I think we are good to go

**Comment by danielmarbach** [comment]
Waiting for 7.1.8 so that the announcement can be combined

**Pull request merged**